### PR TITLE
Hee 261 new service banner

### PR DESCRIPTION
**Pull request by goodvibes**
https://manifesto.atlassian.net/browse/HEE-261

Adds banner to top of page

### CODE REVIEW

**Review by Raq-HEE — COMMENTED** [review]
Hi David - this looks good. Just the readme to do, as it looks like it has the maps component readme.

File: app/assets/components/phase-banner/_index.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@import "../tag/index";
```
Comment (smanivasag) [inline app/assets/components/phase-banner/_index.scss:1]: @goodvibes - Is this a duplicate of `app/assets/components/phase-banner/_phase-banner.scss` ? If so, could we get rid of it ?

Comment (goodvibes) [reply app/assets/components/phase-banner/_index.scss:1]: It's not, the tag .scss handles the CSS around the Alpha / Beta tag in the original phase banner. The tag class in _phase-banner.scss is just tweaking a much larger class.

Comment (smanivasag) [reply app/assets/components/phase-banner/_index.scss]: Thanks @goodvibes for the commit 883c1d26296b133da756d71755ff2d712a6cbb1c to remove `app/assets/components/phase-banner/_index.scss` as `tag` component specific CSS classes have already been included as part of `nhsuk-frontend` (as discussed) 👍

**Review by smanivasag — APPROVED** [review]
Banner tag is still being read as `ALPHA` (rather than `NEW SERVICE` as described in the ticket), but this will be editable within CMS platform when integrated and so shouldn't be an issue.

**Review by Raq-HEE — APPROVED** [review]
Hi David - thanks for the PR. Looks good. Approved.

**Pull request merged**